### PR TITLE
fix: getDeviceId was not working. Added an example on how to use it a…

### PR DESCRIPTION
**Pull request by adricanedo**
fix: getDeviceId was not working. Added an example on how to use it additionally

### CODE REVIEW

**Review by Daamii — APPROVED** [review]

**Comment by frandieguez** [comment]
Be aware that this is a breaking change, you change a callback to a promise so this must be informed in the documention.

Other than that this is approved by me.

File: example/src/App.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import Wayfinding from './examples/wayfinding/Wayfinding';
 import NavigateToPoi from './examples/wayfinding/NavigateToPoi';
 import SelectPoi from './examples/wayfinding/SelectPoi';
 import SelectPoiCategory from './examples/wayfinding/SelectPoiCategory';
+import {DeviceIdentifier} from './examples/sdk/DeviceIdentifier';
```
[the remark this answers is not in the document]
Comment (adricanedo) [reply example/src/App.tsx:26]: Efectivamente, arreglado. Gracias!

**Comment by frandieguez** [comment]
LGTM

File: src/sdk/index.ts [hunk 2]
```diff
@@ -233,13 +233,18 @@ export default class SitumPlugin {
...
    * Returns the device identifier that has generated the location
-   *
-   * @returns void
-   * @throw Exception
    */
   static getDeviceId = () => {
-    return exceptionWrapper<string>(({ onSuccess }) => {
-      RNCSitumPlugin.getDeviceId(onSuccess);
+    return promiseWrapper<string>(({ resolve, reject }) => {
+      RNCSitumPlugin.getDeviceId((response) => {
```
Comment (albasitum) [inline src/sdk/index.ts:239]: Eu vexoo ben, depende de cómo o queiramos sacar hacia o integrador. Podemos non facer ningunha comprobación e simplemente deixar

`  static getDeviceId = () => {
    return promiseWrapper<string>(({ onSuccess }) => {
      RNCSitumPlugin.getDeviceId(onSuccess);
    });
  };`

E que o integrador se encargue de obter o valor dentro da resposta, como no resto das chamadas

**Pull request merged**